Source/Mosa.DeviceSystem/Bitmap.cs

```diff
@@ -20,25 +20,25 @@ public static Image CreateImage(byte[] data)
 			var height = stream.Read32(0x16);
 			var dataSectionOffset = stream.Read32(0xA);
 
-			var image = new Image(width, height);
+			var image = new Image(width, height, bytesPerPixel);
 
-			int x = 0;
-			int y = height - 1;
-
-			for (var i = dataSectionOffset; i < dataSectionOffset + (width * height * bpp); i += bpp)
+			// TODO: Support for other color depths
+			var y = height - 1;
```

Comment:
set y = height

Reply:
It was on height - 1 before. (Like, at the very start)